app/templates/main/search-transferring-body.html

```diff
@@ -26,121 +26,145 @@
-                        </tr>
-                    </thead>
-                    <tbody class="govuk-table__body">
-                        {% for record in results %}
+            <div class="govuk-grid-row">
+                <!-- TABLE -->
+                <div class="govuk-grid-column-two-thirds govuk-grid-column-two-thirds--search-results mobile-table">
+                    <table class="govuk-table" id="tbl_result">
+                        <thead class="govuk-table__head">
                             <tr class="govuk-table__row">
-                                <td class="govuk-table__cell govuk-table__cell--search-results">
-                                    <a href="{{ url_for('main.browse_series', _id=record['series_id']) }}">{{ record["series"] }}</a>
-                                </td>
-                                <td class="govuk-table__cell govuk-table__cell--search-results govuk-table__cell--search-results-no-wrap">
-                                    <a href="{{ url_for('main.browse_consignment', _id=record['consignment_id']) }}">{{ record["consignment_reference"] }}</a>
-                                </td>
-                                <td class="govuk-table__cell govuk-table__cell--search-results">
-                                    <a href="{{ url_for('main.record', record_id=record['file_id']) }}">{{ record["file_name"] }}</a>
-                                </td>
-                                <td class="govuk-table__cell govuk-table__cell--search-results">
-                                    <strong class="{% if record['closure_type'] == 'Open' %}govuk-tag govuk-tag--green{% elif record['closure_type'] == 'Closed' %}govuk-tag govuk-tag--red{% elif record['closure_type'] is none %}{% endif %}">
-                                        {{ record['closure_type'] }}
-                                    </strong>
-                                </td>
-                                <td class="govuk-table__cell govuk-table__cell--search-results">
-                                    {% if record['opening_date'] %}
-                                        {{ record['opening_date'] }}
-                                    {% else %}
-                                        -
-                                    {% endif %}
-                                </td>
+                                <th scope="col"
+                                    class="govuk-table__header govuk-table__header--search-header search__desktop-heading">
+                                    Series
+                                </th>
+                                <th scope="col"
+                                    class="govuk-table__header govuk-table__header--search-header search__mobile-heading">
+                                    Series / File name / Consignment ref
+                                </th>
```

Comment:
why don't you introduce view type of something condition based to show the dynamically

Reply:
View_type cannot account for screen sizes.

Reply:
after discussion it is clear , no need to implement view type